devops/docker/docker-compose-db.yml

```diff
@@ -0,0 +1,61 @@
+version: '3.8'
```

Comment:
I have deleted this file in my previous PR to unify the local build with bootstrap script.
Please add it to the main docker-compose file:
https://github.com/DeloitteDigitalAT/core-banking/blob/master/devops/docker/docker-compose.yml

Reply:
Removed this file and added code to the other